pages/common/pyinfra.md

```diff
@@ -0,0 +1,28 @@
+# pyinfra
+
+> Automates infrastructure at a large scale.
+> More information: <https://docs.pyinfra.com>.
+
+- Execute a command over ssh:
+
+`pyinfra {{target_ip_address}} exec -- {{command_name_and_arguments}}`
+
+- Execute contents of a deploy file on a list of targets:
+
+`pyinfra {{path/to/inventory.py}} {{path/to/deploy.py}}`
+
+- Execute commands on your local machine:
+
+`pyinfra @local exec -- {{command_name_and_arguemnts}}`
+
+or
```

Comment:
We have a linter which enforces a strict format, so unfortunately we can't have an `or` branch here. The format is very much
```markdown
- Example description

`example command`
```